### ISSUE TITLE
Disallow mod structure gen in space station dimids

### ISSUE THREAD

**Issue by zmaster587** [zmaster587/AdvancedRocketry#343]
TiC's slime islands spawn there by default

**Comment by Soyaro666** [comment]
Did you know that, at least in MC 1.7.10, TiC has a config to only spawn slime islands in overworld? And I guess there's a black- and/or whitelist for dims where slime islands are allowed to spawn. just wanted to mention it...

**Comment by Gliese-832-c** [comment]
Yes, the black/whitelist exists, but it would be nicer if you didn't have to configure it and the mod would do it automatically.
